python/lib/cloudutils/serviceConfig.py

```diff
@@ -374,6 +374,8 @@ def addBridge(self, brName, dev):
         if self.syscfg.env.bridgeType == "openvswitch":
             if cfo.getEntry("IPADDR"):
                 cfo.rmEntry("IPADDR", cfo.getEntry("IPADDR"))
+        elif self.syscfg.env.bridgeType == "native":
+            cfo.addEntry("BRIDGE", brName)
```

Comment:
```suggestion
            pass
```
At this point, we're modifying the interface not the bridge